### DISCUSSION TITLE
Pixel 6 with default config, charging doesnt start when battery already low

### DISCUSSION THREAD

**Discussion by mad-tunes** [VR-25/acc#184] [Q&A]
Hi,
I've had a search, sorry if I've missed an answer to this elsewhere.
With my Pixel 6 setting a max charge level works perfectly, but the other end of the scale's causing me problems.

If I let my phones battery drop pretty low (I _think_ below 15%) and connect a mains charger over night, it doesnt charge and ends up nearly dead by the morning.
The only setting I can find that might be involved is resume_capacity, which seems to default to 15%?
(If that isn't the right one, it's whatever shows as 'Resume charging at' in AAC config')

Is that what'll be stopping it charging in this siutation, and is that what it's meant to be doing? (why?)
Should I just set it to 0 to ensure my phone charges once it's already really low?

Thanks

**Comment by VR-25** [comment]
Upgrade to the latest version and see if the issue persists.
`resume_capacity` defaults to 70 in newer versions. If you set it to zero, your battery will never charge while acc daemon is running! (As soon as battery level drops to zero, Android shuts down).
ACC resumes charging when battery level drops to or below resume_capacity.
